src/components/ListTemplate/ColumnTemplates/MatchColumn.js

```diff
@@ -27,14 +27,19 @@ export const MatchColumn = memo(({
 }) => {
   const {
     existingRecordType,
-    existingStaticValueType,
     field,
-    fieldMarc,
-    fieldNonMarc,
+    matchDetails,
   } = record;
 
   const fieldSource = (field || existingRecordType || '').replace(/_/g, ' ');
-  const fieldMatched = (fieldMarc || fieldNonMarc || existingStaticValueType || '').replace(/_/g, ' ');
+  const matchRule = matchDetails[0];
```

Comment:
I don't think this variable is really needed.

Reply:
Done